WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml

```diff
@@ -38,7 +38,16 @@
                 <Style.Setters>
                     <Setter Property="HorizontalAlignment" Value="Stretch" />
                     <Setter Property="MinHeight" Value="68" />
-                    <Setter Property="CornerRadius" Value="{StaticResource OverlayCornerRadius}" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
+                </Style.Setters>
+            </Style>
+
+            <Style
+                x:Key="GridRowLightStyle"
```

Comment:
Ah yes, this is good to do.

Should we rename these so they are consistent?

"GridRowCardStyle" and "GridRowLightCardStyle" or something like that?

And should we move these to App.xaml.cs so they can be re-used across these 2 pages?

Reply:
Sure, do your thing! 👍🏻